webpack.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
         new HtmlWebpackPlugin({
             template: "./src/server/index.html",
             publicPath: "/scripts/",
-            filename: "../server/[name].html",
+            filename: "../server/index.html",
```

Comment:
The [name] placeholder lets us define more entry points. I will often do this when there are different regions of a web application. I don't foresee this in this small example, but it's still a good practice.